pre_install_report/test/test_pre_install_report.py

```diff
@@ -603,40 +601,15 @@ def test_get_k8s_version():
     # initialize the PreCheckPermissions object
     perms = PreCheckPermissions(params)
     perms.set_k8s_git_version(version_string)
-    perms.set_ingress_manifest_file()
-    # check for correct ingress manifest
-    assert(str(perms.get_ingress_file_name() in "hello-ingress-k8s-v118.yaml"))
-
-    # initialize the PreCheckPermissions object
-    perms = PreCheckPermissions(params)
-    perms.set_k8s_git_version(version_string4)
-    perms.set_ingress_manifest_file()
-    # check for correct ingress manifest
-    assert(str(perms.get_ingress_file_name() in "hello-ingress-k8s-v118.yaml"))
-
-    perms.set_k8s_git_version(version_string2)
-    perms.set_ingress_manifest_file()
-    # check for correct ingress manifest
-    assert(str(perms.get_ingress_file_name() in "hello-ingress.yaml"))
 
     # check curren version less than 1.20
     curr_version = semantic_version.Version(str(version_string2))
     assert (curr_version in semantic_version.SimpleSpec('<1.20'))
     assert (curr_version in semantic_version.SimpleSpec('==1.19'))
 
-    perms.set_k8s_git_version(version_string2)
-    perms.set_ingress_manifest_file()
-    # check for correct ingress manifest
-    assert(str(perms.get_ingress_file_name() not in "hello-ingress_invalid.yaml"))
-
-    # initialize the PreCheckPermissions object
-    perms.set_k8s_git_version(version_string3)
-    # check for system exit rc 7
-    try:
-        perms.set_ingress_manifest_file()
-    except SystemExit as exc:
-        assert exc.code == viya_messages.RUNTIME_ERROR_RC_
-        pass
+    # current version is less then 1.19
+    curr_version = semantic_version.Version(str(version_string))
+    assert (curr_version in semantic_version.SimpleSpec('<1.19'))
```

Comment:
Should the "<1.19" value be made into a global variable somewhere? I noticed that value is also used in pre_install_check_permissions.py and a global variable would help with future updates so we don't need to find all the places the minimum version relationship is define.

Reply:
Will look into it making the change.

Reply:
I have added the '<1.19' to the viya_constants.py file which can be imported into any module.

Reply:
Code Updated.

Reply:
Added a <1.20 to the  viya_constants.py file. Completed.